docpad.coffee

```diff
@@ -66,49 +66,49 @@ module.exports =
 
     # The entire schedule
     schedule: [
-      name: '',
-      email: '',
-      photo: '',
-      bio: '',
-      company: '',
+      name: 'Palmer Oliveira',
+      email: 'expalmer@gmail.com',
+      photo: 'themes/yellow-swan/img/palmer.jpg',
+      bio: 'Palmer é um Web Developer entusiasta Javascript que gosta de tentar entender como as coisas funcionam.',
+      company: 'Renove3',
       link:
-        href: '',
-        text: ''
+        href: 'https://expalmer.github.io',
+        text: 'Website'
       presentation:
-        title: 'Palestra 1',
-        description: 'À Confirmar',
+        title: 'Conhecendo o Event Loop no Node',
+        description: 'Quero mostrar de forma lúdica, como funciona a implementação do Event Loop no NodeJs/Browser',
         date: 'Sábado, 18 de Novembro de 2017',
         time: '13:00'
     ,
-      name: '',
-      email: '',
-      photo: '',
-      bio: '',
-      company: '',
+      name: 'Thiago Alves Luiz',
+      email: 'taltk9@gmail.com',
+      photo: 'themes/yellow-swan/img/thiago.jpg',
+      bio: 'Thiago is a front end and JavaScript developer, living in Novo Hamburgo, in Southern Brazil. He started web development over ten years ago building websites for publicity agencies. After this he spent a time developing ERP products at web plataform. He currently specializes in JavaScript applications (specially React applications) and is a Front End Developer at Codeminer 42 in Novo Hamburgo.',
+      company: 'Codeminer 42',
       link:
-        href: '',
-        text: ''
+        href: 'https://www.linkedin.com/in/thiagoalvesluiz/',
+        text: 'Linkedin'
       presentation:
-        title: 'Palestra 2',
-        description: 'À Confirmar',
+        title: 'How to enjoy WebVR right now',
+        description: 'WebVR is an open standard that makes it possible to experience VR in your browser. There are some limits to use this technology in browsers but it is already possible. Lets see how to start VR studies, produce content to use within, some possibilities that it brings and how to enjoy it right now.',
         date: 'Sábado, 18 de Novembro de 2017',
         time: '14:00'
     ,
       name: 'Coffee-Break',
       date: 'Sábado, 18 de Novembro de 2017',
       time: '15:00'
```

Comment:
Aproveita e tira o Coffee-break também, por enquanto.

Reply:
Roger that!